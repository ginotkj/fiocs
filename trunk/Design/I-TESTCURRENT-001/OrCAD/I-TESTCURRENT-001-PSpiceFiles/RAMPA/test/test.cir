** Profile: "RAMPA-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\rampa\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10n 1.5n 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\RAMPA.net" 


.END
