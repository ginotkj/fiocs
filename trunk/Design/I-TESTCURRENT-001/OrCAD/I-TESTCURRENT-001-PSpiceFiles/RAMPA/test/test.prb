[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE I(TRAPEZOIDAL)
MARKERID 2
TRACEADDEXT
INSTANCE 1 TRACECOLOR 3
INSTANCE 1 TRACEWIDTH 4
INSTANCE 1 TRACESYMBOL 0
INSTANCE 1 TRACEUSERMODIFIED
END TRACE I(TRAPEZOIDAL)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
