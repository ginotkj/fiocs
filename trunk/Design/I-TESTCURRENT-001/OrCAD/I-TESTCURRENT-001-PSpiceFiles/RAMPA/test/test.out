
**** 06/07/10 19:13:12 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "RAMPA-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\rampa\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../i-testcurrent-001-pspicefiles/i-testcurrent-001.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 10n 1.5n 
.PROBE V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\RAMPA.net" 



**** INCLUDING RAMPA.net ****
* source I-TESTCURRENT-001
I_I2         N17580 0 DC 0Adc AC 0Aac
+PULSE 0 4m 2n 250p 250p 5n 0
R_R1         N17580 ITEST  1k TC=0,0 
R_R2         0 ITEST  1k TC=0,0 

**** RESUMING test.cir ****
.END

**** 06/07/10 19:13:12 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "RAMPA-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\rampa\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(ITEST)    0.0000  (N17580)    0.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT


    TOTAL POWER DISSIPATION   0.00E+00  WATTS



          JOB CONCLUDED

**** 06/07/10 19:13:12 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "RAMPA-test"  [ D:\Documents\TESIS\fiocs\Design\I-TESTCURRENT-001\OrCAD\i-testcurrent-001-pspicefiles\rampa\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .23
